extension/src/prototypes/view-layers/layerTypeIcons.ts

```diff
@@ -80,6 +81,7 @@ export const layerTypeIcons: Record<LayerType, ComponentType<SvgIconProps>> = {
   [GLOBAL_LAYER]: UseCaseIcon,
   [HIGH_TIDE_RISK_LAYER]: HighTideRiskIcon,
   [INLAND_FLOODING_RISK_LAYER]: InlandFloodingRiskIcon,
+  [RESERVOIR_FLOODING_RISK_LAYER]: InlandFloodingRiskIcon,
```

Comment:
Icon is same with an inland flood model. I'll replace it later after the icon is made.